akka-http-core/src/main/resources/reference.conf

```diff
@@ -460,6 +460,26 @@ akka.http {
     client = {
       # no overrides by default, see `akka.http.client` for default values
     }
+
+    # Add client specific overrides to host-connection-pool. Allows exact and prefixed urls
+    # eg:
+    # per-host-override = [
+    # {
+    #   "akka.io" = { # can use same things as in global `host-connection-pool` section
+    #     max-connections = 10
+    #   }
+    # },
+    # {
+    #   {
```

Comment:
Nesting/indentation is wrong (confusing?) here.

Reply:
I agree it's confusing, but I think it's actually right/required like this, to make sure the list remains ordered...

Reply:
I think here are three opening braces but only two closing?

Reply:
See https://github.com/akka/akka-http/pull/2574/files#r429813471